e2e/rbd.go

```diff
@@ -55,6 +56,18 @@ var (
 	appBlockSmartClonePath = rbdExamplePath + "block-pod-clone.yaml"
 	snapshotPath           = rbdExamplePath + "snapshot.yaml"
 	defaultCloneCount      = 10
+
+	// nolint:gomnd // numbers specify Kernel versions.
```

Comment:
Move this to rbd_helper.go?

Reply:
We will have to import github.com/ceph/ceph-csi/internal/util to  rbd_helper.go too then.